native/jni/core/bootstages.c

```diff
@@ -802,6 +804,7 @@ void late_start(int client) {
 	// Allocate buffer
 	if (buf == NULL) buf = xmalloc(PATH_MAX);
 	if (buf2 == NULL) buf2 = xmalloc(PATH_MAX);
+	if (buf3 == NULL) buf3 = xmalloc(PATH_MAX);
```

Comment:
There should probably be a saner default for allocating the filesystem buffer, but I'm not sure what it is.